lib/cli/commands/ListenOnAdress.ts

```diff
@@ -0,0 +1,28 @@
+import { Arguments } from 'yargs';
+import { loadRPCClient } from '../Command';
+
+export const command = 'listenonadress <address>';
+
+export const describe = 'Detect when a transaction sent to a specific address is confirmed.';
+
+export const builder = {
+  address: {
+    type: 'string',
+    hidden: true,
+  },
+};
+
+export const handler = (argv: Arguments) => {
+  const client = loadRPCClient(argv);
```

Comment:
What about solving this by exposing a gRPC API in Walli instead of loading a RPC in the cli application?

Reply:
I'm not sure what you mean by this, could you elaborate?
Do you mean to create something like this https://github.com/ExchangeUnion/xud/tree/master/proto

Reply:
Yeah I do mean to create something similar to the proto of XUD. But that doesn't have to happen in this PR.